renderer/viewer/sign-renderer/index.ts

```
@@ -135,7 +135,10 @@ export const renderComponent = (
   const textWidths: number[] = []
 
   const renderText = (component: Message, parentFormatting?: Formatting | undefined) => {
-    const { text } = component
+    if (component.text != null && typeof component.text !== 'string') {
```

ESLint

[error] Use '===' to compare with null.

(no-eq-null)